build/configs/rtl8730e/hello/defconfig

```diff
@@ -222,19 +221,27 @@ CONFIG_ARCH_INTERRUPTSTACK=4096
 # CONFIG_BOOT_RUNFROMEXTSRAM is not set
 # CONFIG_BOOT_RUNFROMFLASH is not set
 # CONFIG_BOOT_RUNFROMISRAM is not set
-CONFIG_BOOT_RUNFROMSDRAM=y
+CONFIG_BOOT_RUNFROMDRAM=y
 # CONFIG_BOOT_COPYTORAM is not set
 
+CONFIG_AMEBASMART_BOOT_DRAM=y
+# CONFIG_AMEBASMART_BOOT_NOR is not set
+# CONFIG_AMEBASMART_BOOT_SRAM is not set
+
 #
 # Boot Memory Configuration
 #
+CONFIG_RAM_KREGIONx_START="0x60400000,"
+CONFIG_RAM_KREGIONx_SIZE="4194304,"
+##### Need to crosscheck
```

Comment:
Looks like the defconfig has been reverted back. Could you please rebase to newer code base? Kconfig changes were made and the defconfig was modified in the PR https://github.com/Samsung/TizenRT/pull/5844

Reply:
Hi @abhishek-samsung, we had done updated the defconfig.
Thank You.